h/routes.py

```diff
@@ -115,7 +115,9 @@ def includeme(config):
                      factory='h.traversal.GroupRoot',
                      traverse='/{pubid}')
     config.add_route('api.search', '/api/search')
-    config.add_route('api.users', '/api/users')
+    config.add_route('api.users',
+                     '/api/users',
+                     factory='h.traversal.UserRoot')
```

Comment:
This is the first time we've used a non-default factory on a route without using traversal. _Seems_ to work fine. Any red flags?

Reply:
> This all leads me to ask whether we should just refer to them all as "resources":

As I was doing this work, that started feeling like the intuitive path, for sure. It felt also like if we keep on with our current model, the `roots` and `contexts` modules are going to get enormous.